client/src/component/Title/Title.css

```
@@ -0,0 +1,98 @@
+body {
+background-image: url("Background.png");
+background-repeat: no-repeat;
+    min-height: 92vh;
+    display: flex;
+    flex-direction: column;
+background-attachment: fixed;
+    text-align: center;
+    color: white;
+    font-size: 35px !important;
```

```suggestion
    font-size: 35px;
```

I love using `!important` but you should find where that is being affected and change it so you do not need to use !important, because if you cannot change the font-size here without the !important tag then other files will have the same issue.